tools/device/mkvendor.sh

```diff
@@ -43,6 +43,13 @@ popd > /dev/null
 
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit